### DISCUSSION TITLE
Welcome to HaaConfigurator Discussions!

### DISCUSSION THREAD

**Discussion by Bene2103** [Bene2103/HaaConfigurator#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mario-website** [comment]
Hi, I’m web developer (JavaScript), and I wonder if there is a possibility to write automated script who will convert old json file to new format? As now there is a working code generator for version before 12 - Merlin: https://glumb.github.io/haa-configurator/

**Comment by Bene2103** [comment]
Hello Mario,

sure this would be possible, but thats not the goal of my tool right now. I'm focussing on the primary functions to get the json config for HAA.
But you can always try it yourself, i think this wouldnt be too hard to do it.

**Comment by Bohan-Song** [comment]
looking for help, my english is poor,i can't understand
**For example:**
_I want to configure a switch_

- _GPIO 1 low level switch off_

- _GPIO 1 high level switch on_

- _The switch is connected to GPIO2 through a relay_

How to configure.want a screenshot,thanks

**Comment by Bene2103** [comment]
This should work, configured a Switch with GPIO 1
![image](https://github.com/Bene2103/HaaConfigurator/assets/46553509/0c0af2b4-193f-4390-aaba-8555e4bb9874)
![image](https://github.com/Bene2103/HaaConfigurator/assets/46553509/6f9e2cbe-94df-44d4-924e-405aaf1189ed)

**Reply by Bohan-Song** [reply]
_Thanks~_
